store/mongo/migration_2_0_0.go

```diff
@@ -227,28 +227,27 @@ func (m *migration_2_0_0) Up(from migrate.Version) error {
 		// migrate the documents
 		if collection == DbLimitsColl {
 			for cur.Next(ctx) {
-				id := cur.Current.Lookup(dbFieldID)
-				var currentId string
-				err = id.Unmarshal(&currentId)
-				if err != nil {
-					return errors.Wrap(err, "the id found is un-parsable")
+				id, ok := cur.Current.Lookup(dbFieldID).StringValueOK()
+				if !ok {
+					return fmt.Errorf(
+						"found suspicious limits document: %s",
+						cur.Current.String())
 				}
-				var item bson.D
+				var item bson.M
 				if err = cur.Decode(&item); err != nil {
 					return err
 				}
-				item = append(item, bson.E{
-					Key:   dbFieldName,
-					Value: currentId,
-				})
-				item = item[1:]
-				item = findAndReplace(item, dbFieldID, oid.NewUUIDv4().String())
-
+				item[dbFieldName] = id
+				item[dbFieldTenantID] = tenantID
+				item[dbFieldID] = primitive.NewObjectID()
 				writes = append(writes, mongo.
 					NewReplaceOneModel().
-					SetFilter(bson.D{{Key: dbFieldName, Value: id}}).
+					SetFilter(bson.D{
+						{Key: dbFieldTenantID, Value: tenantID},
```

Comment:
how can this work? we do not have tenant id in the documents.

Reply:
Exactly, but we are doing an upsert so if it doesn't match, it will do an insert.